MaterialComponentsSnapshotTests.podspec

```diff
@@ -32,7 +32,7 @@ module SnapshotPodspecHelper
   end
 
   def self.snapshot_sources
-    base_sources = ["components/private/Snapshot/src/*.{h,m,swift}"]
+    base_sources = ["components/private/Snapshot/src/*.{h,m,swift}", "components/private/Snapshot/src/SnapshotTestCase/*.{h,m,swift}", "components/private/Snapshot/src/SnapshotUtilities/*.{h,m,swift}"]
```

Comment:
Consider using `components/private/Snapshot/src/*/*.{h,m,swift}` so that we don't need to update these in the future if/when we create other extensions.

Reply:
Will do. I didn't know it was possible (I would have preferred `**` syntax, myself).

Reply:
Done.